src/applications/disability-benefits/all-claims/tests/property-names.unit.spec.js

```diff
@@ -0,0 +1,53 @@
+import { expect } from 'chai';
+
+import get from '../../../../platform/utilities/data/get';
+import * as pages from '../pages';
+
+// Some pages have duplicate properties on purpose. These pages add new information to existing
+//  objects and should be ignored.
+const ignoreList = {
+  // propName: ['list', 'of', 'pages'],
+  newDisabilities: ['addDisabilities', 'newDisabilityFollowUp'],
+  serviceInformation: ['federalOrders', 'reservesNationalGuardService'],
+};
+
+describe('Root property names', () => {
+  it('should not be duplicated', () => {
+    const properties = new Map();
+    const duplicatedProperties = {};
+    Object.keys(pages).forEach(pageName => {
+      // Loop through the properties on the schema for a page
+      Object.keys(get(`${pageName}.schema.properties`, pages, {})).forEach(
+        propName => {
+          // Compile a list of duplicate property names
+          if (
+            properties.has(propName) &&
+            !get(propName, ignoreList, []).includes(pageName)
+          ) {
+            if (duplicatedProperties[propName]) {
+              duplicatedProperties[propName].push(pageName);
```

Comment:
So the goal here is to produce a list of every instance of a duplicated property, along with the name of its page for easier referencing?